src/components/Components/__tests__/ComponentListView.spec.tsx

```diff
@@ -2,12 +2,12 @@ import * as React from 'react';
 import '@testing-library/jest-dom';
 import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
 import { render, screen, fireEvent, within } from '@testing-library/react';
-import { WatchK8sResource } from '../../../../dynamic-plugin-sdk';
-import { mockRoutes } from '../../../../hooks/__data__/mock-data';
-import { useApplicationRoutes } from '../../../../hooks/useApplicationRoutes';
-import { useGitOpsDeploymentCR } from '../../../../hooks/useGitOpsDeploymentCR';
-import { ComponentGroupVersionKind } from '../../../../models';
-import { PipelineRunGroupVersionKind } from '../../../../shared';
+import { WatchK8sResource } from '../../../dynamic-plugin-sdk';
```

Comment:
Not important, but I think we no longer need the `dynamic-plugin-sdk.ts` file. All sdk types are now available in `@openshift/dynamic-plugin-sdk-utils`.

Reply:
Trying hard not to mix in other types of rework/refactoring